source/include/FreeRTOS_Routing.h

```diff
@@ -26,16 +26,15 @@
 #ifndef FREERTOS_ROUTING_H
     #define FREERTOS_ROUTING_H
 
-    #include "FreeRTOS.h"
-    #include "FreeRTOS_IP.h"
+/* Global Includes & Definitions. */
+    #include "FreeRTOS_IP_Common.h"
 
-    #if ( ipconfigUSE_DHCP != 0 )
-        #include "FreeRTOS_DHCP.h"
-    #endif
+/* Core FreeRTOS+TCP Includes. */
+    #include "FreeRTOS_IP.h"
 
-    #if ( ipconfigUSE_IPv6 != 0 )
-        #include "FreeRTOS_DHCPv6.h"
-    #endif
+/* Optional FreeRTOS+TCP Includes. */
+    #include "FreeRTOS_DHCP.h"
+    #include "FreeRTOS_DHCPv6.h"
```

Comment:
```suggestion
    #if ( ipconfigUSE_DHCP != 0 )
        #include "FreeRTOS_DHCP.h"
    #endif

    #if ( ipconfigUSE_IPv6 != 0 )
        #include "FreeRTOS_DHCPv6.h"
    #endif
```
Align behavior. But I think we can change `ipconfigUSE_IPv6` to `ipconfigUSE_DHCPv6` here.